### PR TITLE
Fix output casing of prefix dir

### PR DESCRIPTION
**Pull request by mmitche**
The PREFIX dir is upper case, should be refered to as such.  Blocker for building on Linux

### CODE REVIEW

**Comment by mmitche** [comment]
fyi @RussKeldorph

**Comment by RussKeldorph** [comment]
Holy shit that was fast.  I would have asked you to remove the lower-case exclusions in testsFailingOutsideWindows.txt as well, but it’s fine.  I can take care of it.  Thanks!

File: tests/testsFailingOutsideWindows.txt [hunk 1]
```diff
@@ -138,10 +138,10 @@ JIT/Regression/VS-ia64-JIT/V1.2-M02/b22545/b22545/b22545.sh
...
 JIT/Regression/VS-ia64-JIT/V2.0-Beta2/b416776/b416776/b416776.sh
 JIT/Regression/CLR-x86-JIT/V1-M09/b13621/b13621/b13621.sh
-JIT/Directed/prefix/unaligned/1/arglist/arglist.sh
-JIT/Directed/prefix/unaligned/2/arglist/arglist.sh
-JIT/Directed/prefix/unaligned/4/arglist/arglist.sh
-JIT/Directed/prefix/volatile/1/arglist/arglist.sh
+JIT/Directed/PREFIX/unaligned/1/arglist/arglist.sh
+JIT/Directed/PREFIX/unaligned/2/arglist/arglist.sh
+JIT/Directed/PREFIX/unaligned/4/arglist/arglist.sh
+JIT/Directed/PREFIX/volatile/1/arglist/arglist.sh
```
Comment (RussKeldorph) [inline tests/testsFailingOutsideWindows.txt]: These should just be removed.  They are already excluded with this casing at the top of the file.

**Comment by RussKeldorph** [comment]
@mmitche Sorry, I should have been clearer.  If you remove the lines you modified, this should fix #3234.

File: tests/testsFailingOutsideWindows.txt (hunk 1 above)
Comment (mmitche) [inline tests/testsFailingOutsideWindows.txt]: ahh got it, will do.

**Comment by RussKeldorph** [comment]
Closes #3234.

**Pull request merged**